panel/models/__init__.py

```diff
@@ -5,5 +5,5 @@
 files.
 """
 
-from .state import State # noqa
-from .widgets import Audio, FileInput, Player # noqa
+from .state import State  # noqa
+from .widgets import Audio, FileInput, Player  # noqa
```

Comment:
Is there a good reason for these various whitespace changes?  It seemed fine before, to me...

Reply:
+1

Reply:
No there is no good reason. Just auto formating of the editor